api/turing/config/config_test.go

```diff
@@ -693,13 +707,6 @@ func TestConfigValidate(t *testing.T) {
 			},
 			wantErr: true,
 		},
-		"missing vault address": {
-			validConfigUpdate: func(validConfig Config) Config {
-				validConfig.VaultConfig.Address = ""
-				return validConfig
-			},
-			wantErr: true,
-		},
```

Comment:
Would it be better to update/add a test case for cluster config instead just removing existing scenario?

Reply:
I added more test cases with the `required_if` tag.